src/components/Modal/Modal.visual.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import { visualize, story, snap } from 'storybook-snapper';
+import { Modal } from './';
+
+visualize('Modal', () => {
+  story('simple', () => {
+    snap('default props', <Modal isOpen rootElement={document.body} />);
```

Comment:
We should probably add some content to the Modal, or else there won't be much to visualize 😁